### PR TITLE
Update post a job UI to include new fields

### PR DESCRIPTION
**Pull request by ahmedhamodi**
See image below for new fields:
image

### CODE REVIEW

**Comment by ahmedhamodi** [comment]
The tags dropdown is identical to the one in registration (ie. loads from the same file):
image

**Review by arsidada — COMMENTED** [review]
image

1. Looks like you can add more than 3 tags
2. The character limit isn't being enforced in both description and requirements

**Comment by arsidada** [comment]
Should the Country, Region and City be drop downs like they are in registration?

File: src/Components/Dashboard/UserProfile.js [hunk 1]
```diff
@@ -354,18 +357,18 @@ class Landing extends Component {
...
+        title: "Software Engineer",
+        company: "MAX Aspire",
+        country: "Canada",
+        region: "Ontario",
+        city: "Toronto",
         description: "Description 123...",
         requirements: "Requirements 123...",
         posted_by: "ahmed.r.hamodi@gmail.com", // email
         poster_family_name: "Hamodi",
         poster_given_name: "Ahmed",
```
Comment (arsidada) [inline src/Components/Dashboard/UserProfile.js:371]: is there a reason we have default values? should they not just be empty strings?

Comment (ahmedhamodi) [reply src/Components/Dashboard/UserProfile.js:371]: Yes, I will work on that in the second part when I actually set the values from the post a job popup. Right now we are using defaults just so you can populate the jobs easily from the app

**Review by arsidada — APPROVED** [review]

**Pull request merged**